### PR TITLE
Luigi grep py3k fix

### PR DESCRIPTION
**Pull request by gpoulin**
This PR also includes #839

### CODE REVIEW

**Comment by erikbern** [comment]
not sure why it's failing (something with bin_test) but will give it the benefit of the doubt and re-trigger

File: bin/luigi [hunk 1]
```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env python
 
-import luigi
+import sys
+import warnings
+import luigi.cmdline
+
+
+def main(argv):
+    warnings.warn("'bin/luigi' has moved to console script 'luigi'", DeprecationWarning)
```
Comment (Tarrasch) [inline bin/luigi]: @gpoulin, is there any documentation for me how to run this if `bin/luigi` is deprecated?

Comment (gpoulin) [inline bin/luigi]: @Tarrasch, I was again deprecation :P , that was part of #839. I don't think is in the doc, but once you have installed Luigi, the command should be in your path.

``` bash
virtualenv luigi_env
. luigi_env/bin/activate
cd luigi
python setup.py install
luigi someTask
```

Comment (Tarrasch) [inline bin/luigi]: Ah, I've always done tricks with `PYTHONPATH`, maybe I should install luigi the "proper" way. :)

Comment (Tarrasch) [inline bin/luigi]: Ah, this was from when I was on vacation, that's why I don't remember it ^^

**Pull request merged**